tests/func-tests/imx/step-definitions/burning.ts

```diff
@@ -1,14 +1,14 @@
-import { UnsignedTransferRequest } from '@imtbl/core-sdk';
 import {
   IMXClient,
   ImxModuleConfiguration,
   GenericIMXProvider,
   ProviderConfiguration,
+  UnsignedTransferRequest,
 } from '@imtbl/sdk/x';
 import { configuration, StepSharedState } from './stepSharedState';
 
 export class Burning {
-  constructor(protected stepSharedState: StepSharedState) {}
+  constructor(protected stepSharedState: StepSharedState) { }
```

Comment:
is this normal? linting?

Reply:
There's definitely a problem with eslint and prettier configs...I tried fixing them, but that introduced more issues, so I reverted the config changes. This one got through the cracks.